### DISCUSSION TITLE
Add aliases for common cli subcommands

### DISCUSSION THREAD

**Discussion by ameknite** [gleam-lang/gleam#2701] [Ideas & suggestions]
Similar to rust
build -> b
check -> c
docs -> d
run -> r
test -> t

**Comment by lpil** [comment]
Hello! I'm unsure about this one. If a user wants short commands wouldn't it be best for them to use shell aliases? As this would be shorter still.

**Comment by ameknite** [comment]
you are right, I just think these are a good default
